lib/src/option.dart

```diff
@@ -20,13 +20,15 @@ Option newOption(
     bool? splitCommas,
     bool mandatory = false,
     bool hide = false,
+    bool ignoreMandatory = false,
```

Comment:
I would honestly avoid even providing this option for options (only flags). I don't know of any existing use cases for options but I have a feeling any future use cases that might come up would only want to ignore the mandatory warnings if the option was provided a specific value, or something like that.

Reply:
Actually a flag is an `Option`. This is the only structure I have to store any information.

Reply:
Maybe creating a new structure like `Flag` would be a solution.

Reply:
> Actually a flag is an `Option`. This is the only structure I have to store any information.

I understand that Flags are Options, but we don't have to expose the ability to pass this argument for options. You can only create flags/options through these methods, so we can avoid exposing anything for true options.

Reply:
Oh okay. Some maybe instead of the `newOption` function we could use a `newFlag` method.

Reply:
Right, all we have to do is remove this argument from this function and then its only actually supported for flags, afaik at least :).

Reply:
We can do the same thing for the option `negatable` right ?